### ISSUE TITLE
User defined colour palettes

### ISSUE THREAD

**Issue by gchamon** [Guake/guake#1766]
I have looked for similar feature requests, but I couldn't find any. In other terminals, you can install extra colour palettes in a predefined config folder. For instance, konsole has a `.colorscheme` file type that when deployed under `$XDG_DATA_HOME/konsole/` will appear in the selection menu.

I would like to implement something like this for guake. I could extend `palletes.py` to look for these files in a directory and import them. I would suggest something like

```json
{
    "<SCHEME-NAME>": "<SCHEME>"
}
```

For instance, I love snazzy for konsole, and I ported it manually, but I would like to have a file like `~/.config/guake/themes/snazzy.json` with the configuration, so that I could play around with other schemes and go back to my custom scheme without losing it.

I can open a prototypical pull request to illustrate what I want, it that is of interest.

Thanks!

**Comment by lgarczyn** [comment]
Would enjoy this feature. My whole desktop uses arc-dark, and I would enjoy if my terminal would too

**Comment by gchamon** [comment]
@lgarczyn I just migrated to yakuake + KDE. Guake is an awesome project and I would have loved to continue using it, but between problems, lack of features and not hearing from maintainers I just moved on.

**Comment by lgarczyn** [comment]
Thanks, sadly all the alternatives come with their drawbacks. Might try your solution.

**Comment by Davidy22** [comment]
Going back through the log, looks like this is closely related to #220, leaving this here to make the link.

**Comment by gchamon** [comment]
@Davidy22 indeed, user defined and custom pallets are closely related. Good job! Nice to see guake getting some well-deserved love. Kudos to you!
